_includes/docs/user-guide/rule-engine-2-0/re-getting-started.md

```diff
@@ -24,78 +24,178 @@ in related Asset.
 - Make RPC calls based on defined condition.
 - Integrate with external pipelines like Kafka, Spark, AWS services, etc.
 
-## Hello-World Example
-Let’s assume your device is using DHT22 sensor to collect and push temperature to the ThingsBoard. 
+<br/>
+**Let's look at an example.**
```

Comment:
Please leave the '## Hello-World Example' title. The '##" makes an URL anchor that might have been used somewhere.